pkg/sdkserver/workspaces.go

```diff
@@ -0,0 +1,328 @@
+package sdkserver
+
+import (
+	"encoding/json"
+	"fmt"
+	"net/http"
+
+	gcontext "github.com/gptscript-ai/gptscript/pkg/context"
+	"github.com/gptscript-ai/gptscript/pkg/loader"
+)
+
+type workspaceCommonRequest struct {
+	ID                string `json:"id"`
+	WorkspaceToolRepo string `json:"workspaceToolRepo"`
+}
+
+func (w workspaceCommonRequest) getToolRepo() string {
+	if w.WorkspaceToolRepo != "" {
+		return w.WorkspaceToolRepo
+	}
+	return "/Users/thedadams/code/workspace-provider"
```

Comment:
Should be updated before merging

Reply:
Thanks. I thought I got this, but, alas, I forgot.